runtime/doc/eval.txt

```diff
@@ -6360,7 +6360,7 @@ synstack({lnum}, {col})					*synstack()*
 		valid positions.
 
 system({cmd} [, {input}])				*system()* *E677*
-		Get the output of the shell command {cmd} as a |string|.  {cmd}
+		Get the output of the command {cmd} as a |string|.  {cmd}
 		will be run the same as in |jobstart()|.  See |systemlist()|
 		to get the output as a |List|.
```

Comment:
Pushed a couple of commits to update the docs. In the `system()` help I also removed the `Pipes are not used.` line  since that does not seem to be true anymore.